gamemodes/core/account/account_setup-inventory.inc

```diff
@@ -0,0 +1,20 @@
+#include <YSI_Coding\y_hooks>
+
+hook OnMySQLConnected()
+{   
+    static const query[] = "\
+        CREATE TABLE IF NOT EXISTS inv \
+        (\
+            u_id int(11) NOT NULL AUTO_INCREMENT,\
+            ifix tinyint(4) NOT NULL DEFAULT '0',\
+            LifeIns tinyint(4) NOT NULL DEFAULT '0',\
```

Comment:
better names for table, like I've mentioned before.
```
instant_fix
lifetime_insta_fix
```

Reply:
Done removed LifeIns we don't need it yet.